src/components/Support/form/SupportSearch.tsx

```diff
@@ -160,24 +163,26 @@ const MultiSearch: React.FC = () => {
           </form>
         </div>
       </div>
-
-      <div>
+      <div className="absolute left-[50%] top-[80%] z-[10] w-[90%] -translate-x-[50%] md:w-[50%] md:text-[1.5rem] xl:w-[40%]">
         {loading && (
-          <div className="rounded-md absolute left-[50%] top-[100%] z-20 max-h-[250px] w-[90%] -translate-x-[50%] overflow-scroll bg-gray-dark-1 text-[1.3rem] md:w-[50%] md:text-[1.5rem] xl:w-[40%]">
-            Loading...
-          </div>
+          <div className="rounded-md max-h-[250px] w-[90%] -translate-x-[50%] bg-gray-dark-3 text-[1.3rem] md:w-[50%] md:text-[1.5rem] xl:w-[40%]" />
         )}
         {results?.length > 0 && isOpen && !loading && (
           <div ref={dropdownRef}>
-            <ul className="rounded-md absolute left-[50%] top-[100%] z-20 max-h-[250px] w-[90%] -translate-x-[50%] overflow-scroll bg-gray-dark-1 text-[1.3rem] md:w-[50%] md:text-[1.5rem] xl:w-[40%]">
+            <ul className="rounded-md search-results-list z-20 max-h-[250px] overflow-auto rounded-b-[12px] bg-gray-dark-3 p-[12px] text-[1.3rem] shadow-2xl">
```

Comment:
Doesn't seem utilized?

```suggestion
            <ul className="rounded-md z-20 max-h-[250px] overflow-auto rounded-b-[12px] bg-gray-dark-3 p-[12px] text-[1.3rem] shadow-2xl">
```